tests/unit/test.utils.js

```diff
@@ -606,4 +606,105 @@ describe('isCompatible', () => {
       )
     ).toBe(true);
   });
+
+  it('should be compatible if version is behind flag and no other version is available', () => {
```

Comment:
would you mind to briefly describe me the reasoning behind the behavior expected by this set of test cases?

In particular I'm wondering about the following ones:
- compatible with firefox 60, if on all versions it is locked behind a flag
- compatible with firefox 60, if the only version listed is > 60 and it is also locked behind a flag
- incompatible with firefox 60, if on earlier version was locked behind a pref and then it is available without in a later version

Reply:
Right, so this is a bit misleading, because what the rule really cares about in the end is if it's not compatible, because that's when it'll report a warning/error. In the case where it is not officially supported yet there's a different rule that applies no matter the min version set in the manifest.json. The assumption here is that things behind a flag should be caught by that rule.

This also explains the case for why it would be reported as incompatible in the last case. The generic rule thinks this API is supported, however the particular min version the add-on author specified does not yet support the feature, thus this rule springs in action and warns them.

Reply:
ah, I think that I see now, yeah it is a bit misleading while looking at it in isolation (without taking into account that this `isCompatible` check is only going to add a linting error when it detects an explicit incompatibility with the min version specified in the manifest.json).

Thanks for the additional details.